docs/about_knxiot.rst

```diff
@@ -21,29 +24,31 @@ This chapter covers:
 KNX Point API devices
 *********************
 Products that implement the KNX IoT Point API stack use IPv6-based protocols, such as Thread, for communication with the rest of the KNX installation.
-All KNX devices, regardeless of the transmittion methods used, can be easily interconnected within the same system.
+All KNX devices, regardeless of the transmittion methods used, can be easily interconnected.
 
 KNX IoT Point API devices are based on the same interoperability concept as KNX TP (Twisted Pair), KNX PL (Power Line) and KNX RF (Radio Frequency) methods.
-It means that the same semantic data are shared through the transport layer.
-Lower layers of the protocol stack are not KNX-specific - the Thread protocol is used instead.
+It means that the same semantic data are shared through the transport layer as for non-IoT KNX implementations.
+Lower layers of the protocol stack are not KNX-specific — in this case, the Thread protocol is used instead.
 For this reason, a middleware is required for sharing the data between KNX IoT and non-IoT parts of the system.
 
  .. note::
     The middleware functionality could be implemented as a separate device called KNX IoT router.
     It is connected to the same IP network as the KNX IP router.
 
     The KNX IoT router is responsible for transmitting data in both directions:
-    
-    * From KNX IoT Point API devices through the Thread Border Router to other KNX devices through the KNX IP router.
+
+    * From KNX IoT Point API devices through the Thread Border Router and KNX IP routers to other KNX devices.
     * From other KNX devices through the KNX IP, KNX IoT and Thread Border Routers to KNX IoT Point API devices.
- 
-    Moreover, if it is needed and there are enough hardware resources, this feature can be implemented as an additional service on Thread Border Router device.
 
-The diagram below presents a sample of the KNX installation that combines KNX devices using various communication methods within a single system.
+    Moreover, if it is needed, and there are enough hardware resources, this feature can be implemented as an additional service on the Thread Border Router device.
 
- .. figure:: /images/knx_iot_sys_sample.svg
-    :alt: Sample KNX installation consisting of KNX classic and IoT devices.
+The diagram below presents a sample of the KNX installation that combines KNX devices using various communication methods within a single system:
+
+.. figure:: /images/knx_iot_sys_sample.svg
```

Comment:
The diagram's legend to be updated: what is the difference between solid and  dashed line? Is is a connection type, i.e., wireless vs. wired?

Reply:
Good point... solid line means wired connection, dashed line - wireless connection.